docs-source/content/samples/simple/azure-kubernetes-service/troubleshooting.md

```diff
@@ -0,0 +1,334 @@
+   Node Name:      aks-nodepool1-11471722-vmss000001
+   Server Name:    managed-server2
+   State:          RUNNING
+   Cluster Name:   cluster-1
+   Desired State:  SHUTDOWN
+   Server Name:    managed-server3
+   Cluster Name:   cluster-1
+   Desired State:  SHUTDOWN
+   Server Name:    managed-server4
+   Cluster Name:   cluster-1
+   Desired State:  SHUTDOWN
+   Server Name:    managed-server5
+```
+{{% /expand %}}
+
+#### Domain debugging
+
+Some suggestions for debugging problems with Model in Image after your Domain YAML file is deployed are found in the section on [debugging](/weblogic-kubernetes-operator/userguide/managing-domains/model-in-image/debugging/).
+
+#### WSL2 bad timestamp
+
+If you are running with WSL2, you may run into the [bad timestamp issue](https://github.com/microsoft/WSL/issues/4245), which blocks Azure CLI. You may see the following error:
+
+```shell
+$ kubectl get pod
+Unable to connect to the server: x509: certificate has expired or is not yet valid: current time 2020-11-25T15:58:10+08:00 is before 2020-11-27T04:25:04Z
+```
+
+You can run the following command to update WSL2 system time:
+
+```
+# Fix the outdated systime time
+$ sudo hwclock -s
+
+# Check systime time
+$ data
+Fri Nov 27 13:07:14 CST 2020
+```
+
+#### Timeout for the operator installation
+
+You may run into a timeout while installing the operator and get the following error:
+
+```
+$ helm install weblogic-operator kubernetes/charts/weblogic-operator \
+   --namespace sample-weblogic-operator-ns \
+   --set serviceAccount=sample-weblogic-operator-sa \
+   --set "enableClusterRoleBinding=true" \
+   --set "domainNamespaceSelectionStrategy=LabelSelector" \
+   --set "domainNamespaceLabelSelector=weblogic-operator\=enabled" \
+--wait
+Error: timed out waiting for the condition
+```
+
+Make sure you are working with the master branch. Remove the operator and install again.
+
+```bash
+$ helm uninstall weblogic-operator -n sample-weblogic-operator-ns
+release "weblogic-operator" uninstalled
+```
+
+Checkout master and install the operator.
+
+```bash
+$ cd weblogic-kubernetes-operator
+$ git checkout master
+$ helm install weblogic-operator kubernetes/charts/weblogic-operator \
+   --namespace sample-weblogic-operator-ns \
+   --set serviceAccount=sample-weblogic-operator-sa \
+   --set "enableClusterRoleBinding=true" \
+   --set "domainNamespaceSelectionStrategy=LabelSelector" \
+   --set "domainNamespaceLabelSelector=weblogic-operator\=enabled" \
+   --wait
+```
+
+#### WebLogic Image Tool failure
+
+You will get an error running `./imagetool/bin/imagetool.sh` if the docker buildkit is enabled. 
+
+Here is the warning message shown:
+
+```text
+failed to solve with frontend dockerfile.v0: failed to create LLB definition: failed to parse stage name "WDT_BUILD": invalid reference format: repository name must be lowercase
+```
+
+To resolve the error, disable docker buildkit with the following commands and run the `imagetool` command again. 
```

Comment:
disable docker buildkit  -> disable the Docker buildkit

Reply:
Will do.